src/documents/components/ReviewDetail/ReviewVersions.js

```diff
@@ -0,0 +1,213 @@
+import {
+  Button,
+  Header,
+  Icon,
+  Label,
+  Popup,
+  Segment,
+  Table,
+} from 'semantic-ui-react';
+import {Link, withRouter} from 'react-router-dom';
+
+import FileActionButtons from '../FileActionButtons/FileActionButtons';
+import KfId from '../../../components/StudyList/KfId';
+import PropTypes from 'prop-types';
+import React from 'react';
+import {fileSortedVersions} from '../../utilities';
+import {fileTypeDetail} from '../../../common/enums';
+
+/**
+ * Displays data review attached document version list with action buttons
+ */
+const ReviewVersions = ({
+  studyId,
+  reviewNode,
+  updateReview,
+  updateError,
+  setAddFileOpen,
+  downloadFileMutation,
+  canAddFile,
+}) => {
+  const groupType = ['PDA', 'GWO', 'SEQ', 'S3S'];
+  const groupVersions = () => {
+    const versions = reviewNode ? reviewNode.versions.edges : [];
+    var versionGroups = {
```

Comment:
We will probably want to expand these once we get other document types in.

Reply:
Yes, this is a temporary list and I will update it once we have new document types merged in.